### ISSUE TITLE
[bug]: replace or clarify `hybrid` in prompt documentation

### ISSUE THREAD

**Issue by keturn** [invoke-ai/InvokeAI#2500]
https://github.com/invoke-ai/InvokeAI/blob/76b0bdb6f9aa1cb0b12c2f5ea741997b7abe8c63/docs/features/OTHER.md#L68-L74

seen also here:

https://github.com/invoke-ai/InvokeAI/blob/76b0bdb6f9aa1cb0b12c2f5ea741997b7abe8c63/docs/features/PROMPTS.md#L292-L297

Problems:
- People often read this as `hybrid` is a _keyword_ or relevant to the _syntax,_ which, as far as I know, is not the case.
- This is either duplicated content, or two features using the same syntax?
- The description in PROMPTS.md is more accurate, describing the result as "blending concepts." The OTHER.md language is "tell the sampler to invest 25% of its effort," and I wouldn't mind if that were just another metaphor, but "sampler" is a technical term and implies things we are not doing in prompt blending.

The detailed prompt blending example does eventually clarify that "hybrid" is a word here, a part of the prompt introducing a third concept that is neither "blue sphere" or "red cube", but only at the very end. The example should either be clear from the start that it is blending _three_ concepts (and the weight of the third is `1.0` by default), or stick to just two.

https://github.com/invoke-ai/InvokeAI/blob/31146eb7975d3755d23b9819ff11b1a8275c25fe/docs/features/PROMPTS.md#L363-L364

### Additional context

When searching the code for `hybrid` https://github.com/invoke-ai/InvokeAI/search?l=Python&q=hybrid

a number of things come up.

Some of them are related to the inpainting model?

There is this, which _does_ pass more stuff to the sampler and was perhaps the original subject of the "tell the sampler" documentation: https://github.com/invoke-ai/InvokeAI/blob/76b0bdb6f9aa1cb0b12c2f5ea741997b7abe8c63/ldm/models/diffusion/shared_invokeai_diffusion.py#L184

but I don't know if that's still live code. It hasn't come up in the diffusers integration.

**Comment by lstein** [comment]
Good timing. I’m reviewing all the documentation and will fix this ambiguity.

**Comment by keturn** [comment]
the referenced passages are still in the documentation, stalebot

**Comment by Void2258** [comment]
If I am understanding how this works (the syntax is very hard to parse), it functions as 'prompt editing' (I don't like that term but it's standard), as in A1111 syntax [item 1:item 2:% change] to transition a concept?

It would be nice if we could have a better syntax for this operation. Perhaps (item 1, item 2).switch(% change) (since .blend is already used for full prompt blending)? Or the current .blend() could be adjusted to handle single terms instead of needing full prompt strings. Alternation (switch back and forth every iteration rather than after X% of iterations) would be the only case not covered by that syntax.
